WebChannelAPI.md

```diff
@@ -0,0 +1,474 @@
+
+See [Qt's Documentation](https://doc.qt.io/qt-6/qtwebchannel-javascript.html) for more details.
+
+### Quickstart Code Sample
+
+This code block will get you up and communicating with Meld Studio.
+
+```js
+ function connect() {
+    // Currently, Meld Studio will always listen on this address:port.
+    // This may be configurable in the future.
+    const address = "127.0.0.1";
+    const port = 13376;
+
+    var socket = new WebSocket(`ws://${address}:${port}`);
+
+    socket.onopen = function () {
+      // Store this somewhere so that it doesn't get GC'd.
+      const channel = new QWebChannel(socket, function (channel) {
+
+        // Optionally store `meld` for direct access to the Meld Studio API.
+        const meld = channel.objects.meld;
+ 
+        // -------------------------
+        // Example Property Read:
+        // This dumps an object containing all Scenes, Layers, Effects, and Audio
+        // keyed by ID.
+        
+        console.log(meld.session.items);
+
+        // Status properties.
+        
+        console.log(meld.isStreaming);
+        console.log(meld.isRecording);
+
+        // -------------------------
+        // Example Listener:
+        // Subscribing to receive notifications when the Session property changes.
+        
+        meld.sessionChanged.connect(() => {
+          console.log("sessionChanged()");
+        });
+
+        // -------------------------
+        // Example Call:
+        // Will tell Meld Studio to toggle the recording state.
+        
+        meld.toggleRecord();
+      });
+    };
+
+   socket.onclose = function () {
+      console.log("Closed");
+   };
+
+   socket.onerror = function (error) {
+      console.log("Error: ", error);
+   };
+  }
+```
+
+## WebChannel Properties, Signals, and Slots
+
+### Properties
+
+| property    | type     | description                                                                                                                       |
+| ----------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| isStreaming | `bool`   | Indicates active streaming status.                                                                                                |
+| isRecording | `bool`   | Indicates active recording status.                                                                                                |
+| session     | `object` | Represents the current session state with a structured JSON object detailing scenes, tracks, layers, and effects. #session-schema |
+
+### Signals
+
+Signals provide methods for registering `listeners`.
+
+```js
+// Connect a signal callback.
+const ref = meld.signalName.connect((parameter: type) => {
+  // Do something with the information.
+});
+
+// Disconnect the signal callback.
+meld.signalName.disconnect(ref);
+```
+
+1. ## **`gainUpdated(trackId: string, gain: Number, muted: bool)`**
```

Comment:
I would leave off the number. Here, you're using a H2 equivalent underneath an H3, should be H4 aka `####`

Reply:
Fixed in 7d676dc136ad0ef8390ce78eeca8b749bd479623